### PR TITLE
Flux autopilot

### PR DESCRIPTION
**Pull request by aep**

### CODE REVIEW

File: bootstrap/bootstrap.sh [hunk 1]
```diff
@@ -4,12 +4,18 @@ HERE=$(dirname $(readlink -f $0))
...
+# helm repo add cilium https://helm.cilium.io/
 helm install --namespace foundations cilium cilium/cilium --version 1.15.7 --values ${HERE}/cilium-values.yaml
-flux install --namespace foundations
-helm install --namespace foundations foundations  ${HERE}/../charts/foundations --version 0.1.0
 
+kubectl apply -k ${HERE}/../flux
+
+foundations_yaml=$(mktemp)
+kubectl get configmap foundations-config -n foundations -o jsonpath='{.data.foundations\.yaml}' >$foundations_yaml
+helm install --namespace foundations foundations ${HERE}/../charts/foundations --values $foundations_yaml
```
Comment (aep) [inline bootstrap/bootstrap.sh:20]: no this was intentionally bootstrapped empty.

File: charts/foundations/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,8 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.11
+version: 0.1.9
```
Comment (aep) [inline charts/foundations/Chart.yaml:18]:
```suggestion
version: 0.1.12
```

File: charts/foundations/templates/cert-manager.yaml [hunk 3]
```diff
@@ -14,6 +14,7 @@ metadata:
   namespace: foundations
 spec:
   interval: 5m
+  serviceAccountName: foundations-admin
```
[the remark this answers is not in the document]
Comment (aep) [reply charts/foundations/templates/cert-manager.yaml:17]: oh is this how cross-namespace RBAC works? maybe should be called flux-admin or something then? idk

File: bootstrap/bootstrap.sh [hunk 4]
```diff
@@ -4,12 +4,18 @@ HERE=$(dirname $(readlink -f $0))
...
 
 kubectl create namespace foundations
 kubectl apply -f ${HERE}/user-skeleton.yaml
+
+# helm repo add cilium https://helm.cilium.io/
 helm install --namespace foundations cilium cilium/cilium --version 1.15.7 --values ${HERE}/cilium-values.yaml
-flux install --namespace foundations
-helm install --namespace foundations foundations  ${HERE}/../charts/foundations --version 0.1.0
 
+kubectl apply -k ${HERE}/../flux
```
Comment (aep) [inline bootstrap/bootstrap.sh:16]: this doesnt look right. thats without kustomize isnt it? so wrong namespace

File: charts/foundations/templates/cert-manager.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Karitham) [reply charts/foundations/templates/cert-manager.yaml:17]: For the multi-tenancy stuff, we require the use of a serviceAccountName when applying specs.

File: bootstrap/bootstrap.sh (hunk 4 above)
Comment (Karitham) [reply bootstrap/bootstrap.sh:16]: That's kustomize; `-k` is kustomize. The file embeds the namespace (I could extract it too)

File: charts/foundations/templates/cert-manager.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Karitham) [reply charts/foundations/templates/cert-manager.yaml:17]: Yeah I can rename it

**Pull request merged**